components/map/Map.tsx

```diff
@@ -58,11 +58,10 @@ const Map = forwardRef(
     const [location] = useLocation()
     const insets = useSafeAreaInsets()
     const screenCenter = useScreenCenter({ scale: Platform.OS === 'android' })
-    const [selectedPolygon, setSelectedPolygon] = useState<Feature<
-      Geometry,
-      GeoJsonProperties
-    > | null>(null)
-    const [selectedPoint, setMapInterestPoint] = useState<Feature<Point, GeoJsonProperties> | null>(
+    const [selectedPolygon, setSelectedPolygon] = useState<Feature<Polygon, MapUdrZone> | null>(
+      null,
+    )
+    const [selectedPoint, setMapInterestPoint] = useState<Feature<Point, MapInterestPoint> | null>(
```

Comment:
`selectedPoint` is not used, also some constants are left on top. Consider removing the disabled rule `@typescript-eslint/no-unused-vars` and enable it again :)